studio.kx.carla.json

```diff
@@ -8,11 +8,12 @@
     "rename-icon": "carla",
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
         "--socket=wayland",
+        "--socket=fallback-x11",
```

Comment:
no, you really want to disable wayland here. Plugins UI won't work if not running with X11.

Reply:
ah okay thanks, will change it back